app/models/supplejack_api/top_collection_metric.rb

```diff
@@ -0,0 +1,136 @@
+# frozen_string_literal: true
+
+module SupplejackApi
+  # app/models/supplejack_api/top_collection_metric.rb
+  class TopCollectionMetric
+    include Mongoid::Document
+
+    METRICS_LOGGER = Logger.new('log/metrics.log') # run `$ touch log/metrics.log` to create metrics log file
```

Comment:
Do you have to make this file manually each time?

Reply:
i think so. do you know a way of automating this when integrating the api engine into other apps?

Reply:
yes